Webhooks prereqs (#3135)
fishtown-analytics/docs.getdbt.com: website/docs/docs/deploy/webhooks.md

## Prerequisites
- You have a dbt Cloud account that is on the [Team or Enterprise plan](https://www.getdbt.com/pricing/). 

> - You have set up a multi-tenant deployment environment for your dbt jobs. For more information, refer to [Tenancy](/docs/cloud/about-cloud/tenancy). 

Reviewer: You have a multi-tenant deployment in dbt Cloud. For more information, refer to [Tenancy](/docs/cloud/about-cloud/tenancy).

> - You have set up a multi-tenant deployment environment for your dbt jobs. For more information, refer to [Tenancy](/docs/cloud/about-cloud/tenancy). 

Author: ```suggestion
- You have a multi-tenant deployment in dbt Cloud. For more information, refer to [Tenancy](/docs/cloud/about-cloud/tenancy). 
```
